### DISCUSSION TITLE
Convert multiples mp3 to single M4B

### DISCUSSION THREAD

**Discussion by Tanarion40** [advplyr/audiobookshelf#3176] [Q&A] [answered]
Hello.

I read that audiobookshelf has the option to merge multiple mp3 files into a single m4b audiobook. However, I checked all options, and I can't find this tools.
Can someone tell me where exactly I can find it ? (I'm on NAS Qnap)

**Comment by nichwall** [comment, marked as answer]
Edit item -> Tools Tab -> m4b

You will probably want to change the default bit rate, the default is 128k, so if your audiobook has a lower bit rate the m4b book will end up being larger

**Reply by Tanarion40** [reply]
Thank you.
I though would be a "bulk" option :)
